docs/tutorials/Authentication.md

```diff
@@ -0,0 +1,51 @@
+### What is Authentication on Roblox?
+Authentication is pretty simple, it's how Roblox knows who you are and what account you're using. Since Roblox uses FunCaptcha and have no official bot system, developers are forced to use account [**cookies.**](https://support.mozilla.org/en-US/kb/cookies-information-websites-store-on-your-computer) This will allow Roblox to easily identify your bot's account and execute actions.
```

Comment:
Would be nice to have some explicit language that says that we use an **authentication token** which happens to be stored within cookies. And it links to an authentication session.